examples/spotify-api-example/pom.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>Spotify API Example</name>
+    <description>An Apollo example application using the Spotify API</description>
+    <groupId>com.spotify</groupId>
+    <artifactId>spotify-api-example-service</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.spotify</groupId>
+                <artifactId>apollo-bom</artifactId>
+                <version>1.0.0-SNAPSHOT</version>
```

Comment:
Doh, my mistake. Copied this from another example which I guess is wrong as well then.